src/libraries/System.Net.Http/src/System/Net/Http/BrowserHttpHandler/BrowserHttpHandler.cs

```diff
@@ -221,7 +223,7 @@ protected internal override async Task<HttpResponseMessage> SendAsync(HttpReques
 
                 HttpResponseMessage httpResponse = new HttpResponseMessage((HttpStatusCode)status.Status);
 
-                bool streamingEnabled = request.Properties.TryGetValue("WebAssemblyEnableStreamingResponse", out object? streamingEnabledValue) && (bool)(streamingEnabledValue ?? false);
+                request.Options.TryGetValue<bool>(EnableStreamingResponse, out bool streamingEnabled);
```

Comment:
This should be under StreamingSupported if

Reply:
Addressed in recent commits